boards/nxp/imx93_evk/imx93_evk_mimx9352_a55.dts

```diff
@@ -64,6 +65,14 @@
 			gpios = <&gpio2 24 GPIO_ACTIVE_LOW>;
 		};
 	};
+
+	flexcan2_phy: can-phy-tja1057 {
+		compatible = "nxp,tja1057", "can-transceiver-gpio";
+		max-bitrate = <8000000>;
+		standby-gpios = <&gpio_exp0 8 GPIO_ACTIVE_HIGH>;
+		#phy-cells = <0>;
+		status = "disabled";
```

Comment:
Why is this disabled by default?

Reply:
If they are disabled, the tests you just ran have not tested anything significant.

They need to be enabled by default if the board provides them by default. The software drivers etc. will still only be compiled in if the user/application enables `CONFIG_CAN`.

Reply:
ok. these are enabled.